index.js

```diff
@@ -66,6 +68,17 @@ function getPackageManagerImplementation(v) {
       sync: "pnpm install --frozen-lockfile",
       list: (package) => `pnpm list ${package}`,
     },
+    bun: {
+      install: (packages, isDev) =>
+        "bun add " +
+        (isDev ? "--dev " : "") +
+        (isLooseVersion ? "" : "--exact ") +
+        packages,
+      sync: `bun install --frozen-lockfile`,
+      list: (package) => {
+        return `bun pm ls | ${isWindows ? "findstr" : "grep"} ${package}`;
```

Comment:
> would this show them both?

that would require `bun pm ls --all` which returns the entire dependency tree, could grep that too, but it looks a little funky due to the tree 

example with glob-parent in a basic remix app
```sh
bun pm ls --all | rg glob-parent

│           └── glob-parent@5.1.2
│   └── glob-parent@5.1.2
│       └── glob-parent@5.1.2
│   └── glob-parent@5.1.2
├── glob-parent@6.0.2
│       └── glob-parent@5.1.2
│       └── glob-parent@5.1.2
```

Reply:
ok - yeah let's go with the simple one for now.  If it becomes an issue we can expand it to do the full `ls --all` and print the jacked up tree.

I don't expect it to be an issue since I don't expect much else to have `@remix-run/react` as a nested dep.  

Which is different from `react-router-dom` which gets into this scenario because a lot of folks install their own and don't sync it with the internal Remix pinned version (and then the duplicate React contexts cause all sorts of issues):

```
~/me/shopify/repos/playgrounds/temp> npm ls react-router-dom
temp@ /Users/matt/me/shopify/repos/playgrounds/temp
├─┬ @remix-run/react@2.0.0-pre.3
│ └── react-router-dom@6.16.0-pre.0
└── react-router-dom@6.15.0
```